addons/l10n_it_edi/tests/export_xmls/bill_reverse_charge.xml

```diff
@@ -71,6 +71,7 @@
                     Product B, taxed 4%
                 </Descrizione>
                 <Quantita>1.00</Quantita>
+                <UnitaMisura>Pack of 6</UnitaMisura>
```

Comment:
Why are these exported now if they weren't before?

Reply:
Because it was exported based on a condition related to UoM categories (only if not in the units category): https://github.com/odoo/odoo/pull/184131/files/7e45f79d2819529512906619358588155b0fa393#diff-811190e17a6319ef140e8c723545eec04db37d3cad73a7c855f0808aa74131d1L21